src/java_bytecode/java_object_factory.cpp

```diff
@@ -20,6 +20,11 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <linking/zero_initializer.h>
 
+#include <ansi-c/c_types.h>
+#include <ansi-c/string_constant.h>
+
+#include <goto-programs/goto_functions.h>
+
```

Comment:
Ok. (Though I'd suspect that ansi-c/string_constant.{h,cpp} should be moved to util; for c_types.h that's already happening in #731.)